src/test/java/flyingnugget/FlyingNuggetTest.java

```diff
@@ -0,0 +1,50 @@
+package flyingnugget;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class FlyingNuggetTest {
+
+    @Test
+    public void testOne() throws IOException {
```

Comment:
You should follow the naming convention `featureUnderTest_testScenario_expectedBehavior` for more descriptive test names.